scripts/deployment/rinkeby_testnets.json

```diff
@@ -25,9 +27,11 @@
             "l1_messenger": ["OptimismL1Messenger", "0xBa6f1aAF3F909F208b224F3985eCda745FA9101A"],
             "l2_messenger": "OptimismL2Messenger",
             "finalization_time": 604800,
-            "claim_stake": 4700000000000000,
-            "claim_period": 3600,
-            "challenge_period_extension": 3600
+            "request_manager_arguments": {
+                "claim_stake": 0.00047,
```

Comment:
Is this correct?
```
>>> eth_utils.from_wei(4700000000000000, 'ether')
Decimal('0.0047')
```

Reply:
0.00047 is the correct number

probably the other was not intended. But it's not that important since it is on testnet.